src/marker_cache.cpp

```diff
@@ -187,10 +187,20 @@ std::shared_ptr<mapnik::marker const> marker_cache::find(std::string const& uri,
                 }
             }
             //svg.arrange_orientations();
-            double lox,loy,hix,hiy;
-            svg.bounding_rect(&lox, &loy, &hix, &hiy);
-            marker_path->set_bounding_box(lox,loy,hix,hiy);
-            marker_path->set_dimensions(svg.width(),svg.height());
+            double width, height;
+            width = svg.width();
```

Comment:
could this be `const width = ...; const height = ...;` ? 
I do not see any other changes and in my opinion this would be better.

(And same below.)

Reply:
:+1:

Reply:
Thank you, I never leaned C++ properly.